File: app-core/src/main/java/ru/micron/dto/UserDto.java
package ru.micron.dto;

import java.util.UUID;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Accessors(chain = true)
public class UserDto {

  private UUID uuid;
  private String username;
  private String password;
  private String firstName;
  private String lastName;
  private String email;
}

File: app-core/src/main/kotlin/ru/micron/dto/UserDto.kt
package ru.micron.dto

import java.util.*

data class UserDto(
    var uuid: UUID? = null,
    var username: String? = null,
    var password: String? = null,
    var firstName: String? = null,
    var lastName: String? = null,
    var email: String? = null
)
